docs/_static/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("theme.css");
+@import url("styles/furo.css");
```

Comment:
the "copy output" button on jupyter cells is really unclear in darkmode:
![Screenshot_20240429_110526](https://github.com/Ouranosinc/xclim/assets/91751417/feb12ed6-7826-47e0-92d3-e51d1448ebd3)

Reply:
We can modify the CSS using an override: https://sphinx-copybutton.readthedocs.io/en/latest/use.html#modify-the-copy-button-s-style-with-css.

Perhaps we can make this responsive in the conf.py? https://pradyunsg.me/furo/customisation/#light-css-variables-dark-css-variables

Reply:
Alternatively, this button is responsive to mouseover to make it interactive/attention-grabbing: 
![image](https://github.com/Ouranosinc/xclim/assets/10819524/6ee7cd7a-dbbd-473b-a320-2cf414e73edf)